plugin/teksi_wastewater/interlis/gui/interlis_export_settings_dialog.py

```diff
@@ -29,6 +29,12 @@ def __init__(self, parent):
             [config.MODEL_NAME_VSA_KEK, config.MODEL_NAME_SIA405_ABWASSER],
         )
 
+        # Fill orientation selection combobox
+        self.export_orientation_selection_comboBox.clear()
+        self.export_orientation_selection_comboBox.addItem("90.0", "+90°")
+        self.export_orientation_selection_comboBox.addItem("0.0", "0°")
+        self.export_orientation_selection_comboBox.addItem("-90.0", "-90°")
```

Comment:
```suggestion
        self.export_orientation_selection_comboBox.addItem("90°", 90.0)
        self.export_orientation_selection_comboBox.addItem("0°", 0.0)
        self.export_orientation_selection_comboBox.addItem("-90°", -90.0)
```

Here you could add data directly as number so you don't need to convert it back after